Snowflake.Data.Tests/UnitTests/Session/WaitingQueueTest.cs

```diff
@@ -48,22 +50,28 @@ public void TestWaitForTheResourceUntilCancellation()
         public void TestWaitUntilResourceAvailable()
```

Comment:
Let's use Retry instead of this workaround
https://docs.nunit.org/articles/nunit/writing-tests/attributes/retry.html

Reply:
done